cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/exception/CloudImageException.java

```diff
@@ -0,0 +1,16 @@
+package com.sequenceiq.cloudbreak.cloud.exception;
+
+public class CloudImageException extends CloudConnectorException {
```

Comment:
If we want to use this exception specifically for triggering fallback to vhd image in case marketplace image deployment fails, shouldn't we name it as **AzureMktPlaceFallbackException**? This name looks generic and we might accidentally use it for other image exception cases.

Reply:
The logixc to trigger the fallback is subject to change. Overall its a good suggestion, my reasoning with the naming was that the code that cathes it is generic in terms of cloud providers, however I have to check the cloud platform anyway, so I might just go ahead with your proposal in the next change.